packages/spectacle/src/components/code-pane.tsx

```diff
@@ -184,6 +184,7 @@ const CodePane = forwardRef<HTMLDivElement, CodePaneProps>(
     );
   }
 );
+CodePane.displayName = 'CodePane';
```

Comment:
ESLint yelled at me. I think pnpm installed a newer version of `eslint-plugin-react`.

Reply:
Ah yeah this makes sense as `CodePane` is declared as a `forwardRef` anon function.